### PR TITLE
fix: allow signing with smart contract wallets

### PR DESCRIPTION
**Pull request by vm**
Resolves an issue with permit signatures not working with smart contract wallets overloading the signature field.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -5090,7 +5090,7 @@
   resolved "https://registry.yarnpkg.com/@uniswap/token-lists/-/token-lists-1.0.0-beta.30.tgz#2103ca23b8007c59ec71718d34cdc97861c409e5"
   integrity sha512-HwY2VvkQ8lNR6ks5NqQfAtg+4IZqz3KV1T8d2DlI8emIn9uMmaoFbIOg0nzjqAVKKnZSbMTRRtUoAh6mmjRvog==
 
-"@uniswap/universal-router-sdk@1.3.4", "@uniswap/universal-router-sdk@^1.3.4":
+"@uniswap/universal-router-sdk@^1.3.4":
```
Comment (vm) [inline yarn.lock:5093]: this seems wrong

File: package.json [hunk 2]
```diff
@@ -145,7 +145,7 @@
     "@uniswap/sdk-core": "^3.0.1",
     "@uniswap/smart-order-router": "^2.10.0",
     "@uniswap/token-lists": "^1.0.0-beta.30",
-    "@uniswap/universal-router-sdk": "1.3.4",
+    "@uniswap/universal-router-sdk": "^1.3.6",
```
Comment (zzmp) [inline package.json:148]: Why `^`?

**Review by zzmp — APPROVED** [review]
Make sure to also update `widgets`.

File: package.json (hunk 2 above)
Comment (vm) [reply package.json:148]: every other package basically uses ^ and i don't see a downside, especially since we have a yarn.lock

**Pull request merged**